org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JDTDelegateCommandHandler.java

```diff
@@ -79,6 +80,12 @@ public Object executeCommand(String commandId, List<Object> arguments, IProgress
 				case "java.project.import":
 					ProjectCommand.importProject(monitor);
 					return null;
+				case "java.project.resolveSourceUri":

```

Comment:
Since the input is the stacktrace text, the command name such as `java.project.resolveStackTraceLocation` is more descriptive.